test/TTBasic/rlexe.xml

```diff
@@ -536,6 +536,86 @@
       <tags>exclude_dynapogo,exclude_jshost,exclude_snap,exclude_serialized</tags>
     </default>
   </test>
+
+<!-- temp -->
+  <test>
+    <default>
+      <files>promise_MultipleThenCalls.js</files>
+      <compile-flags>-TTRecord=promise_MultipleThenCallsTest -TTSnapInterval=0</compile-flags>
+      <baseline>promise_MultipleThenCallsRecord.baseline</baseline>
+      <tags>exclude_dynapogo,exclude_jshost,exclude_snap,exclude_serialized</tags>
+    </default>
+  </test>
+<!-- temp -->
+
+<!-- 
+temp disabled
```

Comment:
nit: Can you reference some issue number so that we can identify why this is disabled, if we have to look it up later?

Reply:
Not going to check in like this, just letting CI run to see if failures are consistent w/ #5366.   If it passes, I'll clean up & re-enable all those commented out variations.